src/Symfony/Component/Console/Command/Command.php

```diff
@@ -284,6 +284,13 @@ public function setCode($code)
             throw new \InvalidArgumentException('Invalid callable provided to Command::setCode.');
         }
 
+        if (PHP_VERSION_ID >= 50400 && $code instanceof \Closure) {
+            $r = new \ReflectionFunction($code);
+            if (null === $r->getClosureThis()) {
```

Comment:
It's ok I understand now :D , `getClosureThis`  is not documented, so it would be nice if we add a comment that explain why we use it